doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc

```diff
@@ -0,0 +1,21 @@
+[id='create_archetype_project']
+= Creating a project using archetypes
+
+Archetypes are projects that are installed in Apache Maven repositories and contain a specific template structure. You can also generate parameterized versions of the project templates using archetypes. You can use archetypes to create a project in {CENTRAL}. When you create a project, it is added to the Git repository that is connected to your {PRODUCT} installation.
+
+.Prerequisites
+
+* You have created an archetype and listed in the {CENTRAL} *Settings*.
+
+* You have set a default archetype in your space in {CENTRAL}.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects*.
+. Click *Add Project*.
+. Type the project name and description in the *Name* and *Description* fields respectively.
```

Comment:
Enter a project name.... *Description* fields. Remove respectively. Not necessary.